docs/BR.md

```diff
@@ -1756,142 +1754,916 @@ The CA SHALL enforce multi-factor authentication for all accounts capable of dir
+| \ \ \ \ `distributionPoint` | MUST         | The `DistributionPointName` MUST be a `fullName` formatted as described below. |
+| \ \ \ \ `reasons`           | MUST NOT     |                 |
+| \ \ \ \ `cRLIssuer`         | MUST NOT     |                 |
+| \ \ **3**                   | MUST NOT     | `DistributionPoints` that do not conform to the above requirements MUST NOT be present. |
 
-   * `certificatePolicies:policyIdentifier` (Required)
+Table: `fullName` profile
 
-      A Policy Identifier, defined by the issuing CA, that indicates a Certificate Policy asserting the issuing CA's adherence to and compliance with these Requirements.
+| __Field__                           | __Presence__ | __Description__ |
+| ----                                | -            | -----           |
+| `fullName`                          |              |                 |
+| \ \ **1**                           | MUST         | The first `GeneralName` present in `fullName` MUST be of type `uniformResourceIdentifier` |
+| \ \ \ \ `uniformResourceIdentifier` | MUST         | The HTTP URL of the Issuing CA's CRL service for this certificate. |
+| \ \ **2+**                          | MAY          | Additional `GeneralName`s MAY be present. If present, they MUST be of type `uniformResourceIdentifier`. |
+| \ \ \ \ `uniformResourceIdentifier` | MUST         | If present, the scheme of the `uniformResourceIdentifier` MUST be "http". |
+| \ \ **3**                           | MUST NOT     | `GeneralName`s that do not conform to the above requirements MUST NOT be present. |
 
-   The following extensions MAY be present:
+##### 7.1.2.9.3 Signed Certificate Timestamp List
 
-   * `certificatePolicies:policyQualifiers:policyQualifierId` (Recommended)
+If present, the Signed Certificate Timestamp List extension contents MUST be an `OCTET STRING` containing the encoded `SignedCertificateTimestampList`, as specified in [RFC 6962, Section 3.3](https://tools.ietf.org/html/rfc6962#section-3.3).
 
-      `id-qt 1` [RFC 5280].
+Each `SignedCertificateTimestamp` included within the `SignedCertificateTimestampList` MUST be for a `PreCert` `LogEntryType` that corresponds to the current certificate.
 
-   * `certificatePolicies:policyQualifiers:qualifier:cPSuri` (Optional)
+##### 7.1.2.9.4 Subject Key Identifier
 
-      HTTP URL for the Subordinate CA's Certification Practice Statement, Relying Party Agreement or other pointer to online information provided by the CA.
+If present, the `subjectKeyIdentifier` MUST be set as defined within [RFC 5280, Section 4.2.1.2](https://tools.ietf.org/html/rfc5280#section-4.2.1.2). CAs MUST generate a unique `subjectKeyIdentifier` for each unique public key (the `subjectPublicKeyInfo` field of the `tbsCertificate`). For example, CAs MAY generate the subject key identifier using an algorithm derived from the public key, or MAY generate a unique number, such by using a CSPRNG.
```

Comment:
Do you have wording you'd like to suggest? Or were you wanting clarification for intent before doing so?

The intent here was to reflect that 1:many is permitted. At minimum, it requires a unique SKI for each public key, but does not prohibit multiple certificates (containing that public key) to use the same SKI.

The intent around this language is because CryptoAPI prefers AKID:SKID matching over and above certificate subject/issuer matching, and, as of Windows 10, has logic that will cause it to reject a certificate as malicious if it locates a certificate containing a given SKID for a given AKID, and is unable to verify the certificate using that issuing CA's public key. (Technically, it's after it's tried X certificates for a given AKID, and they've all failed, but because of how CAPI caches things, the worst case assumption here is 1)

So it's fine if you have a cert from Org A and Org B, containing the same SPKI and SKID, being used to verify a cert with a given AKID. It's bad if your certs from Org A and Org B contain different SPKIs.

macOS separately does enforcement of AKID/SKID matches, despite RFC 5280 stating such is unnecessary (and RFC 4158 reinforcing this), so this is just trying to avoid these issues.

Reply:
Just to make sure we're on the same page:
- 1 SKID, many SPKIs = 100% bad (breaks Windows, among others)
- 1 SKID per SPKI = Fine. May lead to suboptimal paths (e.g. nameConstrained vs not nameConstrained), but that's a path building issue.
- many SKID, one SPKI = not ideal (can limit macOS chain building)

The goal here is globally unique, and the requirement specifies two ways to achieve that: Using the method described in RFC 5280 (which, yes, uses SHA-1, and is thus weaker than globally unique but acceptable for interoperability) or a CSPRNG.

Which part would you like clarity about? I agree, the wording here (orienting around SKID) makes the inverse table (above) a little less clear, but it was trying to capture "cannot reuse SKIDs unless it's same SPKI", while also trying to capture "globally unique"

Reply:
This language gives two examples of exactly that, so I'm not sure your question. The only way I can make sense of your question is if you believe there should be other methods acceptable; that is, that you believe there are valid cases to ignore the SHOULD/RECOMMENDED of RFC 5280. Given that such divergence has interoperability issues with existing browser software, that seems unlikely, but perhaps it would be clearer if you would elaborate.

The obvious simple, and arguably correct, solution is to MUST the RFC 5280 SHOULD, given that's what existing software expects, but I tried to remain flexible here in reflecting the intent, as well as providing explicitly two concrete methods (from 5280) that meet that intent. So the only issue for an audit would be if a CA is diverging.

Reply:
> My concern is that "globally unique" is describing the end goal and not the mechanism for how uniqueness is achieved.

Agreed. This current approach was reflective of the concerns raised on previous validation calls about restating important parts of RFC 5280 that implementors needed to be aware of.

> I think the best path forward is defining a few acceptable methods for SKID generation without mandating global uniqueness.

Which is what I was thinking the v2 stuff was going to do. I'm fully supportive of doing this in v1, but it sounded like there is opposition.

To make sure I'm understanding: Do you have objections to requiring that CAs use:
1. A CSPRNG
2. The (SHA-1 / SHA-256 / SHA-512) hash of the SPKI (SHA-1 as specified in [RFC 5280, 4.2.1.2](https://datatracker.ietf.org/doc/html/rfc5280#section-4.2.1.2), SHA-256/512 coming from 4.2.1.1)

For **new** CA certificates?

Reply:
Resolving this with https://github.com/sleevi/cabforum-docs/pull/36/commits/feeef8ad5bfb56d0b4263882b91c0a62e8428398